examples/aishell/s0/README.md

```diff
@@ -28,6 +28,8 @@
 | ctc prefix beam search    | 5.17  | 5.81  |
 | attention rescoring       | 4.63  | 5.05  |
 | LM + attention rescoring  | 4.40  | 4.75  |
+| HLG                       | 4.81  | 5.27  |
```

Comment:
How about HLG(k2 LM), so it is easy to understand for the users.

Reply:
Done